source/src/main/java/org/cerberus/core/service/datalib/impl/DataLibService.java

```diff
@@ -720,7 +728,7 @@ private AnswerList<HashMap<String, String>> getDataObjectList(TestDataLib lib, H
                 columnsToHide = getListOfSecrets(lib.getTestDataLibID());
 
                 // CSV Call is made here.
-                responseList = fileService.parseCSVFile(servicePathCsv, lib.getSeparator(), columnList, columnsToHide, execution);
+                responseList = fileService.parseCSVFile(servicePathCsv, lib.getSeparator(), columnList, columnsToHide, ignoreNonMatchedSubdata ? ICsvFileService.IGNORE_NON_MATCHED_COLUMNS : ICsvFileService.DO_NOT_IGNORE_NON_MATCHED_COLUMNS, defaultSubdataValue, execution);
```

Comment:
That could be simplified sending just ignoreNonMatchedSubdata boolean.

Reply:
These variables were used for better readability. The latest commit (#17ba4e0) removes them.